aws-auth-cognito/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
The GitHub rendering of the `consumer-rules.pro` says "No changes." Is that file included, like the other plugins have?

Reply:
It was included but only had the original default comment, not the content like the others had so I went ahead and added the same content as was in the data store one.